arboretum/kubernetes/README.md

```diff
@@ -17,7 +17,76 @@ to include the fetchers and checks from this library in your downstream project.
 
 ## Fetchers
 
-Fetchers coming soon...
+### Cluster Resource
+
+* Class: [ClusterResourceFetcher][fetch-cluster-resource]
+* Purpose: Write the resources of **stand-alone** Kubernetes clusters to the
+evidence locker.  **NOTE:** Do not use this fetcher for managed clusters.
+Instead use the [IBM Cloud cluster list fetcher][ibm-cloud-cluster-list-fetcher].
+* Behavior: Retrieve stand-alone Kubernetes cluster resource data for the provided
+list of clusters.  TTL is set to 1 day.
+* Configuration elements:
+  * `org.kubernetes.cluster_resources.clusters`
+    * Required
+    * List of dictionaries:
+      * `label`: An arbitrary label to specify token in credentials file
+      * `server` A URL as a string to a Kubernetes stand-alone cluster
+  * `org.kubernetes.cluster_resources.types`
+    * Optional
+    * List of resource types as strings
+      * NOTE: For core group API resources, the resource name must be in
+      _plural form_.  See example below.
+      * NOTE: For other "custom" API resources, the resource name must be in the
+      following format: `"APIGROUP/VERSION/NAME"`.  See example below.
+    * Use if looking to override the default of `["nodes", "pods", "configmaps"]`.
+    Otherwise do not include.
+* Expected configuration:
+
+  ```json
+  {
+    "org": {
+      "kubernetes": {
+        "cluster_resources": {
+          "clusters": [
+            {
+              "label": "devcluster1",
+              "server": "https://myserver1:30000"
+            }
+          ],
+          "types": ["nodes", "pods", "apigroup.example.com/v1/mycustom"]
+        }
+      }
+    }
+  }
+   ```
+
```

Comment:
remove one blank line here

Reply:
done. also removed an empty line from the end of the file.